### ISSUE TITLE
Question about Training Loss

### ISSUE THREAD

**Issue by ddz16** [jiangyitong/AutoDIR#9]
Great job! I’d like to ask a question about training loss. In the process of training diffusion model, there are two losses: $L_{LD}$ and $L_{AIR}$ (corresponding to Equation 8 and Equation 10 in the paper). $L_{LD}$ only has gradients for the U-Net of diffusion model, while $L_{AIR}$ has gradients for both U-Net and SCM. I would like to know whether, during the training process of diffusion, only $L_{AIR}$ is used for training, or if the weighted sum of both losses is used for training, or if there are other training methods being employed?

**Comment by ddz16** [comment]
Second, if the loss $L_{LD}$ is used, then what dose $z_t$ represent? Is it obtained by adding sampled noise to the VQVAE feature of the clean image $I_{gt}$?
![image](https://github.com/user-attachments/assets/cfa8b6ea-9ddf-43f7-943b-f5707895a2d3)
